### PR TITLE
Stops testing() abuse ACT2

### PR DESCRIPTION
**Pull request by jknpj**
Moved some of them to warning() and commented the `## WARNING: Invalid f_style for Vox: Square Mustache` warning because it's fucking harmless and super floody.

I hope this will take care of most useless data in our runtime log.

### CODE REVIEW

**Comment by jknpj** [comment]
dang

File: code/game/dna/genes/disabilities.dm [hunk 1]
```diff
@@ -37,8 +37,8 @@
 		M.sdisabilities|=sdisability
 	if(activation_message)
 		to_chat(M, "<span class='warning'>[activation_message]</span>")
-	else
-		testing("[name] has no activation message.")
+//	else
```
Comment (DamianX) [inline code/game/dna/genes/disabilities.dm:40]: This is actually fine, I think we want to know if a gene has no activation message, as was (is?) the case with lactose intolerance. Unless we decide that it's okay for a gene to not have these messages, but I just don't see a reason.

Comment (DamianX) [reply code/game/dna/genes/disabilities.dm:40]: 🤔 perhaps this is better served by a unit test

Comment (jknpj) [reply code/game/dna/genes/disabilities.dm:40]: maybe yes

**Pull request merged**